pkg/client/testclient/fake.go

```diff
@@ -9,19 +9,10 @@ import (
 	"github.com/openshift/origin/pkg/client"
 )
 
-// FakeAction is a wrapper around the Kubernetes FakeAction.
-// Used for faking various actions on resources.
-type FakeAction ktestclient.FakeAction
-
 // Fake implements Interface. Meant to be embedded into a struct to get a default
 // implementation. This makes faking out just the method you want to test easier.
 type Fake struct {
-	// Fake by default keeps a simple list of the methods that have been called.
-	Actions []FakeAction
-	Err     error
-	// ReactFn is an optional function that will be invoked with the provided action
-	// and return a response.
-	ReactFn ktestclient.ReactionFunc
+	*ktestclient.Fake
```

Comment:
I'm not sure timestamps will have the resolution we need to deterministically sort requests that occur very quickly in sequence. If you have to touch every place we add actions anyway, might be better to delegate to an AddAction function. Then we have greater flexibility to make that function add sequence numbers, or timestamps, or append to a shared actions slice, etc